src/components/newsletter/selected_users.js

```diff
@@ -0,0 +1,64 @@
+export default function SubscriberList (props){
+    const value = []
+    
+    if(props.subscribers != undefined) {
+      props.subscribers[props.item_name]?.forEach((v) => {
+        // filter here
+        if(String(v.subscribe) == props.is_subscribed){
```

Comment:
filtering the email list at the front end instead of the backend as discussed on 02/03